{{cookiecutter.project_slug}}/requirements/base.txt

```diff
@@ -6,7 +6,7 @@ wheel==0.30.0
 
 
 # Conservative Django
-django==1.11.8 # pyup: <2.0
+django==2.0 # pyup: <2.0
```

Comment:
I think we can drop this pyup comment, or change it to `<2.1`?

Reply:
Done, dropped it.